tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/runinterminal/ProjectBasedRunInTerminalTest.java

```diff
@@ -0,0 +1,48 @@
+package org.ballerinalang.debugger.test.adapter.runinterminal;
+
+import org.ballerinalang.debugger.test.utils.DebugTestRunner;
+import org.ballerinalang.debugger.test.utils.DebugUtils;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+/**
+ * Test class to test the runInTerminal feature for project based sources.
+ */
+
+public class ProjectBasedRunInTerminalTest {
```

Comment:
Shall we also include a negative test case just to ensure that the debug server is not responding to invalid/unsupported terminal names?

Reply:
+1. Updated.